### ISSUE TITLE
UI.AlertNotification style regression (also, does not accept class names)

### ISSUE THREAD

**Issue by pmeskers** [pivotal-cf/pivotal-ui#158]
The UI.AlertNotification used to create a bell icon with a size of font-size of h4. But now icons can only be sized via `fa-h{size}`, and the UI.AlertNotification does not do this, so it's much smaller than it used to be. This is a breaking change/regression for us.

Also, we can't add the class ourselves, because the component doesn't accept class names.

**Comment by gpleiss** [comment]
https://www.pivotaltracker.com/story/show/96041150

**Comment by gpleiss** [comment]
@pmeskers now that we've added the `fa-h{size}` classes, would you be able to address this regression yourselves and make a PR?

cc/ @stubbornella

**Comment by bebepeng** [comment]
@gpleiss, @stubbornella  Would you like us to still hard code in the size through use of the new class or should the class be passed in? Personally, I think the bell is meant to be a certain size by design, but I could be swayed otherwise.

**Comment by stubbornella** [comment]
Imo, you should make the bell a specific component that doesn't by default
require utils. E.g.

.dropdown-notifications{
Bell specific styles here
}

**Comment by gpleiss** [comment]
Seconded. Maybe the fa-h2 size can be a default, but I think it should be
customizable.

**Comment by stubbornella** [comment]
I'm going to close this issue since we unblocked you, I'll open a new issue for making the bell work without modifiers.
